ProcessLib/LIE/SmallDeformation/SmallDeformationProcessData.h

```diff
@@ -94,6 +94,10 @@ struct SmallDeformationProcessData
 
     // mesh property for fracture aperture
     MeshLib::PropertyVector<double>* _mesh_prop_b = nullptr;
+    MeshLib::PropertyVector<double>* _mesh_prop_w_n = nullptr;
+    MeshLib::PropertyVector<double>* _mesh_prop_w_s = nullptr;
+    MeshLib::PropertyVector<double>* _mesh_prop_fracture_stress_shear = nullptr;
+    MeshLib::PropertyVector<double>* _mesh_prop_fracture_stress_normal = nullptr;
```

Comment:
I'd try to use either `_n` and `_s` or `_normal` and `_shear`, but not to mix the both.